app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+
+  def update_comment_counter
+    posts.increment!(:commentsCounter)
+  end
```

Comment:
- [ ] OPTIONAL
It is a nice idea to use one of the Rails Active Callbacks `after_save` to call the `update_comments_counter method every time a new like has been saved to database, then make the `update_comments_counter` method private, similar to:

```ruby
 # app/models/comments.rb
  after_save :update_comments_counter

  private

  def update_comment_counter
    author.increment!(:comments_counter)
  end
```